### PR TITLE
Fix video capture test to support rotated variations of the resolution.

### PR DESCRIPTION
**Pull request by andresusanopinto**
Fix issue #271.

### CODE REVIEW

File: samples/web/content/testrtc/js/camtest.js [hunk 1]
```diff
@@ -134,13 +139,10 @@ CamCaptureTest.prototype = {
     if (info.isMuted === true) {
       reportError('Camera reported itself as muted.');
     }
-    if (info.videoWidth !== info.mandatoryMinWidth) {
-      reportError('Incorrect captured width.');
+    if (!ResolutionMatchesIndependentOfRotation(info.videoWidth, info.videoHeight,
+                                               info.mandatoryMinWidth, info.mandatoryMinHeight)) {
```
Comment (KaptenJansson) [inline samples/web/content/testrtc/js/camtest.js]: Incorrect indentation.

File: samples/web/content/testrtc/js/camtest.js [hunk 2]
```diff
@@ -56,6 +56,11 @@ function CamCaptureTest() {
   this.video.setAttribute('muted','');
 }
 
+function ResolutionMatchesIndependentOfRotation(a_width, a_height, b_width, b_height) {
```
Comment (KaptenJansson) [inline samples/web/content/testrtc/js/camtest.js]: Lower case on function name.

**Comment by KaptenJansson** [comment]
LGTM with nits.

File: samples/web/content/testrtc/js/camtest.js (hunk 1 above)
Comment (andresusanopinto) [inline samples/web/content/testrtc/js/camtest.js]: Done.

File: samples/web/content/testrtc/js/camtest.js (hunk 2 above)
Comment (andresusanopinto) [inline samples/web/content/testrtc/js/camtest.js]: Done.

**Pull request merged**